SmartDeviceLink/SDLTemperature.h

```diff
@@ -0,0 +1,29 @@
+//
+//  SDLTemperature.h
+//
+
+#import "SDLRPCMessage.h"
+#import  "SDLTemperatureUnit.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SDLTemperature : SDLRPCStruct
+
+-(instancetype) initWithUnit: (SDLTemperatureUnit) unit andValue:(NSNumber<SDLFloat> *) value;
```

Comment:
formatting, `and` is not preferred in most cases. should probably be `initWithUnit:value:`

The value parameter should probably be a `float` and converted into an `NSNumber` in the intializer.